### ISSUE TITLE
breaking an end frame portal provocates Minecraft to freeze (1.9 and 1.9.4)

### ISSUE THREAD

**Issue by Hypsellis** [MinecraftForge/MinecraftForge#2940]
I create this issue because I noticed that the ender protal frames behaved them really strangely.
When I try to destroy an ender portal frame with an ender eye inside, Minecraft totally freezes, then if I wait, Windows tels me that Minecraft stopped to work and try to find a solution, then Minecraft just crashes without any crash reports.
When I create an ender portal in creative, and I put inside the protal and put all the eyes like the precedent versions, nothing happens, the end portal doesn't create it, and switching between the creative and survival mode doesn't change anything.
I can confirm now that this issue comes from Forge since there is no problems with Minecraft Vanilla, and the bug happens even if no mods are installed, only Forge. Switch between different versions of Forge doesn't seem to change anything.

**Comment by diesieben07** [comment]
Logs.

**Comment by Hypsellis** [comment]
http://pastebin.com/AXy82f8c
It's the full log from the loading of minecrat and of a random world, then I placed an ender protal frame with an ender ee inside and broke it, and the game freezes and crashs. I know what the error code -805306369 means, but it's not linked to my memory, I have 16 Go of memory and I gave 12 Go of memory to this Minecraft instance, so it's not possible that I lack of memory.

**Comment by TomasKS** [comment]
You're not giving Java 12 GB in that log, you're giving it 1 GB. "-Xms512m **-Xmx1024m**". 

tl:dr: Probably not a Forge problem.

Using the vanilla launcher I created a fresh minecraft profile with the latest Forge from the Forge site (1940) and no other mods in it's own separate folder, it ran just fine even with only 1GB and allthough it did swallow every bit of RAM almost instantly it never crashed no matter what i did with end portals. I created several worlds, both survival and creative, and built, activated, went through and destroyed End portals in as many different combinations as I could think of without the client crashing. In fact, destroying any single part of the 9 actual portal parts seemed to have no effect at all..the portal remains active until you destroy the very last of the portal part. 

As for failing to actiavte the portal, there's a trick to building an End Portal, you have to place the final ender eye in a block that you placed while standing inside the ring or it wont activate.



Here's my specs from the dump in the logfile:

-- System Details --
Details:
	Minecraft Version: 1.9.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 364676448 bytes (347 MB) / 657457152 bytes (627 MB) up to 954728448 bytes (910 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx1024m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13440 Compatibility Profile Context 16.200.1013.0' Renderer: 'AMD Radeon R9 200 Series'

**Comment by diesieben07** [comment]
http://hopper.minecraft.net/help/exit-code/code-805306369/

You ran out of memory.

**Comment by Actuarius** [comment]
@diesieben07 closed this issue

**Comment by Hypsellis** [comment]
I give 12 Go to Minecraft, if you don't trust me look that.
![7d96417a-25ff-11e6-96f8-566e690a4c50](https://cloud.githubusercontent.com/assets/17103411/15788640/c918819a-29c8-11e6-892a-d63d523c88dd.PNG)
I curruntly build my own modpack, and I have something like 100 mods to load, and I need 8 Go for that. If I had only 1 Go for loading it, Minecraft would even not not starting and would crash.

**Comment by laz2727** [comment]
No you didn't.

And, btw, 12? 4 gigs are already overkill.

**Comment by shadowfacts** [comment]
Your crash log says you have it set to a maximum of 1 GB. Logs don't lie, you have it set to a maximum of 1 GB. As @laz2727 said, allocating 12 gigabytes of memory is overkill. Use something reasonable like 3-4.

**Comment by Hypsellis** [comment]
![capture](https://cloud.githubusercontent.com/assets/17103411/15789323/c8187bc6-29cb-11e6-81bb-e7d7b42da37c.PNG)

**Comment by KnightMiner** [comment]
Try increasing the PermGen box

**Comment by Hypsellis** [comment]
Thanks, it resolved the problem of incorrect values for the Xms and Xmx, now it displays correctly the values I put. But it still doesn't prevent the bug to happen. :(

**Comment by diesieben07** [comment]
Post a log where it did not run out of memory.

**Comment by Hypsellis** [comment]
It doesn't change anything, the values are now correct, but the same bug still happens. I forgot to precise too that in fact Minecraft freezes eternally, the bug happens when I force Minecraft to stop on MultiMC, a windows opens it and Java tells me that it doesn't answer more, and I get the -805306369 error code. http://pastebin.com/vrY5Z3na

**Comment by williewillus** [comment]
The error code is from mmc killing Minecraft. That's not what's causing the freeze.

**Comment by LexManos** [comment]
Take this to the forums this is not the place to troubleshoot user errors.
